strapi/src/components/sections/contacts-section.json

```diff
@@ -0,0 +1,34 @@
+{
+  "collectionName": "components_sections_contacts_sections",
+  "info": {
+    "displayName": "Contacts section",
+    "description": ""
+  },
+  "options": {},
+  "attributes": {
+    "title": {
+      "type": "string"
+    },
+    "description": {
+      "type": "richtext"
+    },
+    "hasBackground": {
+      "type": "boolean"
+    },
+    "contacts": {
+      "type": "component",
+      "repeatable": true,
+      "component": "blocks.contact-card",
```

Comment:
It's not ideal that we loose consistent order how we show phones/emails/address.